app/mainAppWindow/index.js

```diff
@@ -511,16 +509,16 @@ function assignSelectSourceHandler() {
 	};
 }
 
-async function handleOnIncomingCallCreated() {
+async function handleOnIncomingCallCreated(e, data) {
 	if (!incomingCallCommandProcess && config.incomingCallCommand) {
-		let commandParts = config.incomingCallCommand.match(commandSplitRegex);
-		incomingCallCommandProcess = spawn(commandParts.shift(), commandParts);
+		const commandArgs = [...config.incomingCallCommandArgs, data.caller];
+		incomingCallCommandProcess = spawn(config.incomingCallCommand, commandArgs);
 	}
 }
 
 async function incomingCallCommandKill() {
```

Comment:
this should be renamed to 
```suggestion
async function incomingCallCommandTerminate() {
```